packages/desktop-client/src/components/reports/SaveReport.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+
+import ExpandArrow from '../../icons/v0/ExpandArrow';
+import Button from '../common/Button';
+import Menu from '../common/Menu';
+import MenuTooltip from '../common/MenuTooltip';
+import Text from '../common/Text';
+import View from '../common/View';
+
+export function SaveReportMenuButton() {
+  let [menuOpen, setMenuOpen] = useState(false);
+
+  const onMenuSelect = async item => {
+    switch (item) {
+      case 'save':
+        setMenuOpen(false);
+        break;
+      case 'clear':
+        setMenuOpen(false);
+        break;
+      default:
+    }
+  };
+
+  function SaveReportMenu({ onClose }) {
+    return (
+      <MenuTooltip width={150} onClose={onClose}>
+        <Menu
+          onMenuSelect={item => {
+            onMenuSelect(item);
+          }}
+          items={[
+            ...[
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏unnecessary object spread usage